client/src/components/SolutionViewer/ProjectModal.tsx

```diff
@@ -15,20 +16,19 @@ const propTypes = {
   //     path: PropTypes.string
   //   })
   // ),
```

Comment:
I cannot make a suggestion for all of this file.

Please delete this comment - this migration is what the `TODO` is referring to.